ci/azure/posix.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       versionSpec: '$(python.version)'
 
   - script: |
-      pip install pytest pytest-cov pytest-mock requests-mock pytest-timeout pytest-azurepipelines pytest-rerunfailures pytest-remotedata
+      pip install pytest==7.0.1 pytest-cov pytest-mock requests-mock pytest-timeout pytest-azurepipelines pytest-rerunfailures pytest-remotedata
```

Comment:
This fixes an unrelated issue.  Probably we'll be able to unpin pytest again once this is resolved: https://github.com/tonybaloney/pytest-azurepipelines/issues/57

Reply:
Downstream issue is resolved.

Reply:
Thanks @tonybaloney!